_posts/2015-03-13-InfoEducatie-Online-2015.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "InfoEducație - Ediția Online 2015"
+date:   2015-03-13 23:16:02
+author: "Cîtea Alexandru"
+categories: stiri
+---
+
+A fost lansat InfoEducație Online 2015!
+
+De anul trecut, a început o nouă tradiție în istoria acestui concurs, și anume, ediția online. A doua ca număr, ediția online de anul acesta își deschide porțile devreme, înscrierile începând de pe data de 18 martie și se încheie pe 18 aprilie.
```

Comment:
Dacă e gerunziu la `incepand` ar trebui gerunziu si `incheindu-se`. Cum e prea complicat, propun sa spargem propozitia in 2.

```
De anul trecut, a început o nouă tradiție în istoria acestui concurs, și anume, ediția online. A doua ca număr, ediția online de anul acesta își deschide porțile devreme. Înscrierile încep de pe data de 18 martie și se încheie pe 18 aprilie.
```